src/Files/Type/MigrationFile.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ *
+ * EPV :: The phpBB Forum Extension Pre Validator.
+ *
+ * @copyright (c) 2014 phpBB Limited <https://www.phpbb.com>
```

Comment:
Even though this is a new file, ie new IP, that was conveived in 2016? Normally a copyright goes to when something was created, assuming you are copyrighting the document itself.